test/404_test.js

```diff
@@ -18,7 +18,8 @@ describe('404', () => {
         done();
     });
 
-    it('valid html', (done) => {
+    it('valid html', function(done) {
+        this.timeout(5000);
```

Comment:
I'd pull the `5000` out in to a variable set in `test_helpers.js`, but I won't block on it.

Reply:
All right, I'll try to do it tomorrow.